web/src/modules/topic/components/Node/EditableNode.styles.tsx

```diff
@@ -62,7 +64,7 @@ const options = {
 
 /* some copied from https://github.com/wbkd/react-flow/blob/147656b22f577bb4141664d000e62ada9b490473/src/theme-default.css#L42-L77 */
 export const NodeDiv = styled("div", options)<NodeDivProps>`
-  width: ${nodeWidth}px;
+  width: clamp(200px, 16rem, ${nodeWidth}px);
```

Comment:
unfortunately the table still doesn't fit 4 columns at 1280px screen width. but I think our node width can be significantly decreased - it's 16rem right now, but I think ~11rem still looks good
![image](https://user-images.githubusercontent.com/13872370/231834389-be987b6d-90cc-4993-ad82-9b682c6d970c.png)

(I think `EditableNode#NodeDiv width` and `NodeCell#StyledDiv min-width` both need to be adjusted)